freckle-app/library/Freckle/App/Test/Hspec/AnnotatedException.hs

```diff
@@ -0,0 +1,22 @@
+module Freckle.App.Test.Hspec.AnnotatedException
+  ( unwrapAnnotatedHUnitFailure
```

Comment:
I might be wrong, but I _think the_ convention for libraries providing hooks is to name the module appropriately and expose only `hook`, then expect users to use it qualified.

```suggestion
module Freckle.App.Test.Hspec.Hooks.UnwrapAnnotatedException
  ( hook
```

:shrug: 

One concrete benefit would be the case where this is the only hook they need, you'd only need to re-export it:

```hs
module SpecHook
  ( module Freckle.App.Test.Hspec.Hooks.UnwrapAnnotatedException
  ) where

import Freckle.App.Test.Hspec.Hooks.UnwrapAnnotatedException
```

How common that is, I do not know.

Reply:
I can't find anything on Hackage's Hoogle resembling this pattern.